demo/README.md

```diff
@@ -87,43 +81,12 @@ TBD - Create Keyvault solution with various secrets and env vars:
 - Various AAD Security env vars
 - UI env vars (if we get dynamic injection supported)
 
-### Deploy load balancer services
-`kubectl apply -f ui-service.yaml && kubectl apply -f ui-service.yaml`
-This step is currently needed so we can hardcode the `api` IP and expose the `UI` layer.
-
-### Create docker images
```

Comment:
We should keep this for the UI images and tell people to deploy their own due to the limitation of client-side code baking in the variables at build time.